app/javascript/components/concept-map/ConceptMap.tsx

```diff
@@ -68,6 +68,7 @@ export const ConceptMap = ({
                     slug={slug}
                     name={concept.name}
                     web_url={concept.web_url}
+                    tooltip_url={concept.tooltip_url}
```

Comment:
Could we use camelCase here?

```suggestion
                    tooltipUrl={concept.tooltip_url}
```